### PR TITLE
Fix error page messages

### PR DESCRIPTION
**Pull request by nadiyafomenko**
fixed the display of various error messages on the error page

### CODE REVIEW

File: src/translations/errorpage.translations.js [hunk 1]
```diff
@@ -1,10 +1,44 @@
...
-    title: 'Щось пішло не так...',
-    toHomepage: 'На головну'
-  },
-  1: {
-    title: 'Something went wrong...',
-    toHomepage: 'Go to homepage'
-  }
+  BUSINESS_TEXT_NOT_FOUND: [
+    { value: 'текст бізнесу не знайдений' },
+    { value: 'business text not found' }
```
Comment (TorskyM) [inline src/translations/errorpage.translations.js:4]: I think the message has to start with a capital letter

File: src/translations/errorpage.translations.js [hunk 2]
```diff
@@ -1,10 +1,44 @@
...
+  PRODUCT_NOT_FOUND: [
+    { value: 'Товар не знайдено' },
+    { value: 'Product not found' }
+  ],
+  ORDER_NOT_FOUND: [
+    { value: 'Замовлення не знайдено' },
+    { value: 'Order not found' }
+  ],
+  PATTERN_NOT_FOUND: [
+    { value: 'Паттерн не знайдено' },
```
Comment (TorskyM) [inline src/translations/errorpage.translations.js:28]: Use `Гобелен` here

**Review by TorskyM — APPROVED** [review]

File: src/translations/errorpage.translations.js (hunk 1 above)
Comment (nadiyafomenko) [reply src/translations/errorpage.translations.js]: fixed

File: src/translations/errorpage.translations.js (hunk 2 above)
Comment (nadiyafomenko) [reply src/translations/errorpage.translations.js]: fixed

**Review by INR4GE — APPROVED** [review]

**Pull request merged**